libs/langchain-community/src/vectorstores/pgvector.ts

```diff
@@ -609,12 +613,14 @@ export class PGVectorStore extends VectorStore {
    * @param query - Query vector.
    * @param k - Number of most similar documents to return.
    * @param filter - Optional filter to apply to the search.
+   * @param includeEmbedding Whether to include the embedding vectors in the results.
    * @returns Promise that resolves with an array of tuples, each containing a `Document` and its similarity score.
    */
   async similaritySearchVectorWithScore(
     query: number[],
     k: number,
-    filter?: this["FilterType"]
+    filter?: this["FilterType"],
+    includeEmbedding?: boolean
```

Comment:
Changing the default signature (even adding params) makes migrations a lot trickier - can we factor this out into a private method?

Sort of like we did with Supabase:

https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-community/src/vectorstores/supabase.ts#L280

Reply:
@jacoblee93 thanks for the review. Have made the suggested changes.